bot/proxyviews.py

```diff
@@ -0,0 +1,41 @@
+from urllib.parse import urljoin
+
+import requests
+from django.conf import settings
+from django.http import StreamingHttpResponse
+from django.views import View
+
+
+class ChatBotProxy(View):
+    def get(self, request, *args, **kwargs):
+        path = request.path
+        base_url = urljoin(settings.CHAT_BOT_URL, path)
+        params = request.GET.dict()
+
+        # adding bot auth key and freva conf
+        params["auth_key"] = settings.CHAT_BOT_AUTH_KEY
+        params["freva_config"] = settings.CHAT_BOT_FREVA_CONFIG
+
+        try:
+            upstream_response = requests.get(base_url[:-1], params=params, stream=True)
+            upstream_response.raise_for_status()
+        except requests.RequestException as e:
+            return StreamingHttpResponse(
```

Comment:
While I was dealing with broken `BOT_AUTH_KEY`s, I had the pleasure to meet this Exception here and I think we need a few adjustments:

- replace `502` by the actual status_code. It seems that it is stored in `e.response.status_code`
- I think it would make things easier if we would deal with a JSON. My initial idea would be to replace the error string by something like 

```
                json.dumps(
                    {
                        "variant": "ServerError",
                        "content": f"Error while connecting to the external API: {str(e)}",
                    }
                ),
```

But I am not sure. In other parts of the django backend we simply create errors like `{"error": "error message"}`, but if it helps you to keep the structure of the rest feel free to alter it. Keep in mind, that this error message will also appear for other bot-requests like `availablebots`